workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -18,34 +18,86 @@ All possible configuration options are available in the
 [Interface API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_interface)
 and [Network API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_network).
 
-> **Note:** Currently, the only supported network type is `pod`.
-
 ## Backend
 
 Network backends are configured in `spec.networks`. A network must have a
 unique name. Additional fields declare which logical or physical device the
 network relates to.
 
+Each network should declare its type by defining one of the following fields:
+
+| Type | Description |
+|--|--|
+| `pod` | Default Kubernetes network |
+| `multus` | Secondary network provided using Multus |
+
+### pod
+
 A `pod` network represents the default pod `eth0` interface configured by
-cluster SDN solution that is present in each pod.
+cluster network solution that is present in each pod.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
+```
+
+### multus
+
+It is also possible to connect VMIs to secondary networks using
+[Multus](https://github.com/intel/multus-cni). This assumes that multus is
+installed accross your cluster and a corresponding
+`NetworkAttachmentDefinition` CRD was created.
 
-> **Note:** Currently, the only supported network type is `pod`. In the future,
-> more network types may be supported.
+The following example defines a network which uses the [ovs-cni
+plugin](https://github.com/kubevirt/ovs-cni), which will connect the VMI to Open
+vSwitch's bridge `br1` and VLAN 100. Other CNI plugins such as ptp, bridge,
+macvlan or Flannel might be used as well. For their installation and usage refer
+to the respective project documentation.
+
+First the `NetworkAttachmentDefinition` needs to be created. That is usually
+done by an administrator. Users can then reference the definition.
+
+```yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ovs-vlan-100
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
```

Comment:
From where do the administrator get cniVersion, how the administrator know this version should be changed?
Or this is some static version that never changed?

Reply:
This is something related to every specific cni plugin.
every plugin can support different versions.